AzureMonitorAgent/agent.py

```diff
@@ -417,13 +417,18 @@ def enable():
         ("azuremonitoragentmgr", False)
     ])
 
+    # Set traceFlags in publicSettings to enable mdsd tracing. For example, the EventIngest flag can be enabled via "traceFlags": "0x2"
+    flags = ""
+    if public_settings is not None and "traceFlags" in public_settings:
+        flags = "-T {} ".format(public_settings.get("traceFlags"))
```

Comment:
shouldn't it be {0}?

Reply:
[The index is not necessary on Python 3.1+](https://docs.python.org/3/library/string.html#formatstrings). "The positional argument specifiers can be omitted for [str.format()](https://docs.python.org/3/library/stdtypes.html#str.format), so '{} {}'.format(a, b) is equivalent to '{0} {1}'.format(a, b)."